### PR TITLE
Cloud documentation

### PR DESCRIPTION
**Pull request by spencer-skylight**
# Description
This contains documentation for the cloud module.

# Additional Changes
`geospatial-tutorial.md` has been updated to reflect the correct naming for abstract classes (`BaseGeocodeClient`, `FhirBaseGeocodeClient`), and import from the correct package (`phdi.geospatial.core`, `phdi.fhir.geospatial.core` respectively).

### CODE REVIEW

**Review by bamader — CHANGES_REQUESTED** [review]
Just a couple of typos, but the content looks good.

File: tutorials/cloud-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Tutorials: The Cloud Module
+This guide serves as a tutorial overview of the functionality available in both `phdi.cloud` and `phdi.fhir.cloud` packages. It covers concepts such as credential managers, cloud storage, and FHIR export file downloads.
+
+## The Basics: Cloud Managers and Clients
+The goal of the `phdi.cloud` and `phdi.fhir.cloud` modules is to provide a cloud platform-agnostic way to interact with common cloud services that have similar, but differing implementations across cloud providers. Doing provides a way for users other building blocks who need to leverage general cloud functionality to do so in a cloud platform-agnostic way.
```
[the remark this answers is not in the document]
Comment (spencer-skylight) [reply tutorials/cloud-tutorial.md]: Updated both.

File: tutorials/cloud-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        '''
+    
+    list_objects()
+        '''
+        List objects within a container.
+        '''
+```
+
+## FHIR Export Download
+Azure FHIR server places files full of FHIR resources in blob storage during [FHIR bulk data exports](http://hl7.org/fhir/uv/bulkdata/export/index.html). To simplify download of these files, you can directly use the completed export job's status. FHIR export and related poll responses are described more detail in the [transport tutorial](transport-tutorial.md).
```
[the remark this answers is not in the document]
Comment (spencer-skylight) [reply tutorials/cloud-tutorial.md]: I reworded this a bit to make it clearer.  Does it look better to you?

**Review by bamader — APPROVED** [review]
LGTM!

**Pull request merged**